source/index.ts

```diff
@@ -353,17 +358,19 @@ export default class Conf<T extends Record<string, any> = Record<string, unknown
 		}
 	}
 
-	private _encryptData(data: string | Buffer): string {
+	private _encryptData(data: string | Uint8Array): string {
 		if (!this.#encryptionKey) {
-			return data.toString();
+			return isUint8Array(data) ? uint8ArrayToString(data) : data.toString();
```

Comment:
This should be simpler and faster:

```suggestion
			return typeof data === 'string' ? string : uint8ArrayToString(data);
```

Reply:
Good point!